process_manager/tables.py

```diff
@@ -26,6 +27,16 @@
     "<a href=\"{% url 'process_manager:logs' record.uuid %}\">LOGS</a>"
 )
 
+header_checkbox_hyperscript = "on click set .row-checkbox.checked to my.checked"
+
+row_checkbox_hyperscript = """
+on click
+if <input.row-checkbox:not(:checked)/> is empty
+  set #header-checkbox.checked to true
```

Comment:
In plain(er) English, this means that if the list of rows that are not checked is empty, then the header checkbox should be checked - as all the rows are checked - right?

Reply:
Correct.